test/functional/element_creation.js

```diff
@@ -61,6 +63,36 @@ describe('element creation', () => {
       expect(el.getAttribute('data-foo')).to.equal('Hello');
       expect(el.getAttribute('data-bar')).to.equal('World');
     });
+
+    it('should return created node', () => {
```

Comment:
Split these into separate tests. To the extent possible, each test should test exactly one thing. You can wrap the related tests in a describe block.

It probably doesn't make sense to put this particular feature test under 'element creation' as it should also apply for updates.